pkg/api/types.go

```diff
@@ -3312,6 +3314,31 @@ const (
 	TLSCertKey = "tls.crt"
 	// TLSPrivateKeyKey is the key for the private key field in a TLS secret.
 	TLSPrivateKeyKey = "tls.key"
+
+	// SecretBootstrapToken is used during the automated bootstrap process (first
+	// implemented by kubeadm). It stores tokens that are used to sign well known
+	// ConfigMaps. They may also eventually be used for authentication.
+	SecretTypeBootstrapToken SecretType = "bootstrap.kubernetes.io/token"
```

Comment:
@kubernetes/api-approvers Adds a new secret type.  I'm pretty sure this was already discussed, so a link from @jbeda may expedite things here.